docs/user/how-to/create-an-adapter.rst

```diff
@@ -1,6 +1,14 @@
 Creating an Adapter
 ===================
 
+.. warning::
+    This is somewhat depreciated in favour of using a `ComposedAdapter` which
+    already exists in the tickit library. An example of this being set up can
+    be found in the tutorial for :doc: `creating a simple simulation. <creating-a-simple-device-simulation>`_
+
+    **To do**: Look to alter or remove this content.
```

Comment:
Maybe this page should just be about adding a composed adapter to the device created by the previous how-to?

Reply:
Already done.